### ISSUE TITLE
Error Message on right click list-item (archives-list)

### ISSUE THREAD

**Issue by SebTM** [borgbase/vorta#1294]
**Describe the bug**
Uncaught exception, please file a report with this text at
https://github.com/borgbase/vorta/issues/new
Traceback (most recent call last):
  File "/nix/store/wrh1ihzaw9sclfh8nydvsv06c1n0mrk0-vorta-0.8.4/lib/python3.9/site-packages/vorta/views/archive_tab.py", line 163, in archiveitem_contextmenu
    menu.addAction(self.bMount.icon(), self.bMount.text(),
AttributeError: 'ArchiveTab' object has no attribute 'bMount

**To Reproduce**
On Archives Tab, make right click on a list item

**Environment (please complete the following information):**
- OS: NixOS
- Vorta version: 0.8.4
- Installed from: Nixpkgs (PR) preparing update for 0.8.4

**Additional context**
```
2022-05-11 22:36:13,437 - vorta.i18n - DEBUG - Loading translation succeeded for ['en', 'en-US', 'en-Latn-US'].
2022-05-11 22:36:13,511 - vorta.views.source_tab - DEBUG - Added item number 0 from 1
2022-05-11 22:36:13,512 - vorta.views.source_tab - DEBUG - Added item number 1 from 2
2022-05-11 22:36:13,512 - vorta.views.source_tab - DEBUG - Added item number 2 from 3
2022-05-11 22:36:13,513 - vorta.views.source_tab - DEBUG - Added item number 3 from 4
2022-05-11 22:36:13,629 - root - INFO - Using NullNetworkStatusMonitor NetworkStatusMonitor implementation.
2022-05-11 22:36:13,676 - vorta.borg.jobs_manager - DEBUG - Add job for site default
2022-05-11 22:36:13,677 - vorta.borg.jobs_manager - DEBUG - Start job on site: default
2022-05-11 22:36:13,689 - vorta.borg.borg_job - INFO - Running command /nix/store/g8qq7qayhdp26pxhw7x5z40j66nvwwbh-borgbackup-1.2.0/bin/borg --version
2022-05-11 22:36:14,022 - vorta.borg.jobs_manager - DEBUG - Finish job for site: default
2022-05-11 22:36:14,022 - vorta.borg.jobs_manager - DEBUG - No more jobs for site: default
2022-05-11 22:36:14,023 - vorta.scheduler - DEBUG - Refreshing all scheduler timers
2022-05-11 22:36:14,026 - vorta.scheduler - INFO - Setting timer for profile 1
2022-05-11 22:36:14,026 - vorta.scheduler - DEBUG - Scheduling next run for 2022-05-11 23:27:58.887098
2022-05-11 22:36:14,029 - vorta.scheduler - INFO - Setting timer for profile 2
2022-05-11 22:36:14,030 - vorta.scheduler - DEBUG - Scheduling next run for 2022-05-11 23:14:31.833984
2022-05-11 22:36:19,879 - root - CRITICAL - Uncaught exception, file a report at https://github.com/borgbase/vorta/issues/new
Traceback (most recent call last):
  File "/nix/store/wrh1ihzaw9sclfh8nydvsv06c1n0mrk0-vorta-0.8.4/lib/python3.9/site-packages/vorta/views/archive_tab.py", line 163, in archiveitem_contextmenu
    menu.addAction(self.bMount.icon(), self.bMount.text(),
AttributeError: 'ArchiveTab' object has no attribute 'bMount'
```

Does not happen for me for 0.8.3, maybe related to the new beautiful design changes :)

**Comment by m3nu** [comment]
Already fixed in ‘master’ branch and will be released as small bugfix version at the end of the week.

**Comment by real-yfprojects** [comment]
Fixed in #1285.
